configure.ac

```diff
@@ -129,6 +158,23 @@ if test x$ac_cv_asm_symver_support = xyes; then
 	AC_DEFINE([HAVE_SYMVER_SUPPORT], 1, [assembler has .symver support])
 fi
 
+AC_CACHE_CHECK(
```

Comment:
I think the use of AC_CACHE_CHECK is just for extra style points here.  You typically only use AC_CACHE_CHECK in m4 that can get "executed" repeatedly.  I.e., if the same chunk of m4 can get repeatedly inserted in configure, the code inside AC_CACHE_CHECK will only be executed once.

This chuck of code will only be executed once, so I wouldn't bother using AC_CACHE_CHECK -- just call AC_LINK_IFELSE/etc. directly.

Also, use the same always-define-HAVE_ALIAS_ATTRIBUTE structure that we talked about for the other boolean #defines -- i.e., have the result of AC_LINK_IFELSE set some shell variable to 0 or 1 and then AC_DEFINE_UNQUOTED to define HAVE_ALIAS_ATTRIBUTE.